packages/bundle-source/src/zip-base64.js

```diff
@@ -7,7 +7,9 @@ import url from 'url';
 import fs from 'fs';
 import os from 'os';
 
-import { makeAndHashArchive } from '@endo/compartment-mapper/archive.js';
+import { parserForLanguage as transformingParserForLanguage } from '@endo/compartment-mapper/archive-parsers.js';
+import { parserForLanguage as transparentParserForLanguage } from '@endo/compartment-mapper/import-parsers.js';
+import { makeAndHashArchive } from '@endo/compartment-mapper/archive-lite.js';
```

Comment:
suggestion: feels like we should add some more proper top-level exports to `@endo/compartment-mapper`, but that's just what _I'd_ do.

Reply:
Much of this is in service of manual tree shaking: allowing the consumer to depend only on what they need. XS in particular is sensitive about Babel, so being able to opt-out depending on which side of compartment-mapper you’re using is key.